ColorPicker/SwatchButton.js

```diff
@@ -24,6 +24,7 @@ import componentCss from './SwatchButton.module.less';
  *
  * @class SwatchButtonBase
  * @memberof agate/ColorPicker
+ * @extends agate/Button.ButtonBase
```

Comment:
Is it private component?

Reply:
No, It's public.

Reply:
But I cannot see SwatchButton on the document ,
SwatchButton link dose not works on the ColorPicker

Reply:
Ah I got to know there is on ColorPicker.